extensions/hibernate-reactive/runtime/src/main/java/io/quarkus/hibernate/reactive/runtime/FastBootHibernateReactivePersistenceProviderResolver.java

```diff
@@ -6,19 +6,24 @@
 import javax.persistence.spi.PersistenceProvider;
 import javax.persistence.spi.PersistenceProviderResolver;
 
+import io.quarkus.hibernate.orm.runtime.HibernateOrmRuntimeConfig;
+
 final class FastBootHibernateReactivePersistenceProviderResolver implements PersistenceProviderResolver {
 
-    private static final List<PersistenceProvider> HARDCODED_PROVIDER_LIST = Collections
-            .singletonList(new FastBootHibernateReactivePersistenceProvider());
+    private final List<PersistenceProvider> persistenceProviders;
+
+    public FastBootHibernateReactivePersistenceProviderResolver(HibernateOrmRuntimeConfig hibernateOrmRuntimeConfig) {
```

Comment:
Very clever! I expected you'd have to bypass the JPA bootstrap, but this seems more effective.